cmake/helper_functions.cmake

```diff
@@ -0,0 +1,68 @@
+#ign_generate_conf()
+#Generate conf/*.yaml files for projects
+macro(ign_generate_conf)
+    #Used only for internal testing.
+    set(ign_library_path "${CMAKE_BINARY_DIR}/test/lib/$<CONFIG>/ruby/ignition/cmd${IGN_DESIGNATION}${PROJECT_VERSION_MAJOR}")
+
+    #Generate a configuration file for internal testing.
```

Comment:
Eh, comments should have a space between the `#` and the body. By remove space i meant the extra lines.